packages/app/src/App.tsx

```diff
@@ -225,7 +226,24 @@ function App(props: { flameFromQuery?: FlameFunction[] }) {
               </Card>
             )}
           </For>
-
+          <Card class={ui.addFlameCard}>
+            <button
+              class={ui.addFlameButton}
+              onClick={() => {
+                setFlameFunctions((draft) => {
+                  draft.push({
```

Comment:
Nit: Since touching the code, perhaps extract this default new transform to a const, maybe we can even create a module for such things, Defaults.tsx or something like that, would make some parts of code much cleaner and avoids magic numbers all over the place

Reply:
extracted to a const above, lets find a place for it when we refactor App